rockcraft/models/project.py

```diff
@@ -279,20 +276,17 @@ def _validate_entrypoint_service(
     def _validate_entrypoint_command(
         cls, entrypoint_command: str, info: pydantic.ValidationInfo
     ) -> str | None:
-        if info.data["entrypoint_service"]:
+        if info.data.get("entrypoint_service", {}):
```

Comment:
```suggestion
        if info.data.get("entrypoint_service", None):
```

The type of entrypoint_service is `str | None`, so we should stick to that to not confuse ourselves